packages/client/components/TaskFooterIntegrateMenu.tsx

```diff
@@ -91,18 +89,16 @@ const TaskFooterIntegrateMenu = (props: Props) => {
       isViewerIntegrated.hasGitHub,
       isViewerIntegrated.hasAtlassian
     )
-    const label = 'Push with your credentials.'
```

Comment:
I was a bit confused by this - is it informing users that they can push to Jira or GitHub with their user credentials instead of team credentials? Or is it letting them know that they can push a task straight to Jira/GitHub? Either way, I felt that removing the full stop was a tiny bit more friendly